src/plugins/data_source_management/public/components/data_source_menu/data_source_menu.test.tsx

```diff
@@ -153,4 +153,16 @@ describe('DataSourceMenu', () => {
     );
     expect(container).toMatchSnapshot();
   });
+
+  it('should render no data source', () => {
+    component = shallow(
```

Comment:
let's not expose this component from data source menu

Reply:
removed